.github/workflows/release.yml

```diff
@@ -147,7 +147,7 @@ jobs:
           name: vscode-codeql-extension
 
       - name: Azure User-assigned managed identity login
-        uses: azure/login@v2
+        uses: azure/login@a457da9ea143d694b1b9c7c869ebb04ebe844ef5 # v2.3.0
```

Comment:
Will dependabot update the "v2.3.0" comment too?

Reply:
Yes, it should: https://github.blog/changelog/2022-10-31-dependabot-now-updates-comments-in-github-actions-workflows-referencing-action-versions/